### PR TITLE
activity

### PR DESCRIPTION
**Pull request by andygray**

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -52,8 +54,7 @@ const store = new Vuex.Store({
     KnownOriginDigitalAssetV1: null,
     KnownOriginDigitalAssetV2: null,
 
-    activity: [],
-    activityStarted: false,
+    KnownOriginDigitalAssetV2MainBlockNumber: 6270484
```
Comment (jamesmorgan) [inline src/store/index.js]: I'll use this as well 👍

File: src/store/modules/activity.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import Vue from 'vue';
+import * as actions from '../actions';
+import * as mutations from '../mutation';
+
+const activityStateModule = {
```
Comment (jamesmorgan) [inline src/store/modules/activity.js]: 👍 better to keep seperate at first

File: src/store/modules/activity.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  getters: {},
+  mutations: {
+    [mutations.SET_ACTIVITY] (state, events) {
+      state.activityStarted = true;
+      Vue.set(state, 'activity', events);
+    }
+  },
+  actions: {
+    [actions.ACTIVITY]: function ({commit, dispatch, state, rootState}) {
+      if (!state.activityStarted) {
```
Comment (jamesmorgan) [inline src/store/modules/activity.js]: Is this needed as you are using a get?

File: src/store/modules/activity.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        rootState.KnownOriginDigitalAssetV2.deployed()
+          .then((contract) => {
+
+            let mintedEvent = contract.Minted({}, {
```
Comment (jamesmorgan) [inline src/store/modules/activity.js]: Looking good, you could also add in `EditionCreated` events as well - would give a nice blend of creation and purchases

File: src/store/modules/kodaV2.js [hunk 5]
```diff
@@ -133,32 +133,32 @@ const contractStateModule = {
     },
   },
   mutations: {
-    [mutations.SET_ACCOUNT_TOKENS](state, tokenAndEditions) {
+    [mutations.SET_ACCOUNT_TOKENS] (state, tokenAndEditions) {
```
Comment (jamesmorgan) [inline src/store/modules/kodaV2.js]: Looks like we are using a different formatter

File: src/store/modules/activity.js (hunk 3 above)
Comment (andygray) [reply src/store/modules/activity.js]: nope - I'll remove

File: src/store/modules/kodaV2.js (hunk 5 above)
Comment (andygray) [reply src/store/modules/kodaV2.js]: yeah

**Pull request merged**